docs/tables/aws_s3_multi_region_access_point.md

```diff
@@ -0,0 +1,78 @@
+# Table: aws_s3_multi_region_access_point
+
+Amazon S3 Multi-Region Access Points provide a global endpoint that applications can use to fulfill requests from S3 buckets located in multiple AWS Regions. You can use Multi-Region Access Points to build multi-Region applications with the same architecture that's used in a single region, and then run those applications anywhere in the world. Instead of sending requests over the congested public internet, Multi-Region Access Points provide built-in network resilience with acceleration of internet-based requests to Amazon S3.
```

Comment:
```suggestion
Amazon S3 Multi-Region Access Point provides a global endpoint that applications can use to fulfill requests from S3 buckets located in multiple AWS Regions. You can use Multi-Region Access Points to build multi-region applications with the same architecture that's used in a single region, and then run those applications anywhere in the world. Instead of sending requests over the congested public internet, Multi-Region Access Points provide built-in network resilience with the acceleration of internet-based requests to Amazon S3.
```